docs/users-guide/start.md

```diff
@@ -17,5 +17,6 @@
 *   [Creating SQL Templates](13-sql-parameters.md)
 *   [Viewing X-ray reports](14-x-rays.md)
 *   [Getting alerts](15-alerts.md)
+*   [Customizing drill-through](16-customizing.drill-through.md)
```

Comment:
Should we still be linking to this in the users guide?

Reply:
No, good catch!